internal/cloud/realm/secrets.go

```diff
@@ -0,0 +1,42 @@
+package realm
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/10gen/realm-cli/internal/utils/api"
+)
+
+const (
+	valuesPathPattern = appPathPattern + "/values"
+)
+
+// Value is a value or secret stored in a Realm application
+type Value struct {
+	ID           string `json:"_id"`
+	Name         string `json:"name"`
+	LastModified int64  `json:"last_modified"`
+	Secret       bool   `json:"from_secret"`
+}
+
+func (c *client) FindValues(app App) ([]Value, error) {
```

Comment:
I used `Value` for the variable name and `FindValues` for the function name since the endpoint retrieves all values. I maintained this throughout the list command as well. Wanted to get thoughts on this since I felt secret just another type of value in this scenario and I am displaying both

Reply:
yeah, I like `Value` here for consistency with how we're calling it on the api/server side